src/share/classes/com/sun/javatest/regtest/exec/RegressionScript.java

```diff
@@ -1168,6 +1173,10 @@ Map<String, String> getTestProperties()  {
         if (disablePreview()) {
             p.put("test.enable.preview", "false");
         }
+        if (isTimeoutsEnabled()) {
+            long seconds = properties.getDefaultTimeout(getTestDescription().getFile()).toSeconds();
+            p.put("test.timeout.default.seconds", Long.toString(seconds));
```

Comment:
Is this the properties that get advertised to the application's test code? I'm guessing it is because I see that the `tag-spec` documentation has been updated to mention that this property will be made available to the test code. 

Is there a reason why the test code would be interested in this default value? As far as I can see we don't expose the final evaluated timeout value nor do we expose any other default configuration values (like the `defaultExecMode`) as properties to the test code. So it might be better not to expose this at least until it is really necessary.

Reply:
Hello Christian, I had forgotten that the test timeout factor gets exposed to test code as a property. Having said that, the test timeout factor is the final determined value unlike this proposed default timeout property which is a default value and may not even be the actual timeout that's ultimately used by jtreg.

Reply:
As discussed offline, this PR won't export that property. It can be done in a follow-up work.